source/blog/embertimes-template.md

```diff
@@ -8,7 +8,9 @@ responsive: true
 
 <SAYING-HELLO-IN-YOUR-FAVORITE-LANGUAGE> Emberistas! 🐹
 
+<goodbits-only>
```

Comment:
Does this one work like a HTML comment on the page?

Reply:
No...just wasn't sure how to specify "template for Goodbits, but not the blog". Is a template on goodbits.com only a better way?

Reply:
Ah okay I see! Yes sure, I like your idea to just add it to goodbits - once we have it in one edition we can just copy over the template each week. Would you like to remove this section from the blog template then?

Reply:
Removed it here, so this PR should be good to go.

I am not 100% sure what is the best way to create a template in Goodbits / create a new Goodbits (the couple times I did it from scratch it was a little time consuming). Maybe we could pair on Goodbits / take a video of you Goodbits-ing if possible? I can also help more in Goodbits so know this process better...